.github/workflows/test.yml

```
@@ -36,9 +40,11 @@ jobs:
           pip install gh-release-install
           # shellcheck disable=SC2002
           tool_count=$(cat "${{ github.workspace }}/config/tools.yml" | yq -p yaml -o json | jq -c '.[] | (if .repo == null then empty else . end ) | .repo' | wc -l)
+          repo_count=$(cat "${{ github.workspace }}/config/repos.yml" | yq -p yaml -o json | jq -c '.[] | (if .repo == null then empty else . end ) | .repo' | wc -l)
+          total_count=$(expr $tool_count + $repo_count)
```

Because actionlint does not love you, but you're being perfectly reasonable

```suggestion
          # shellcheck disable=SC2002
          repo_count=$(cat "${{ github.workspace }}/config/repos.yml" | yq -p yaml -o json | jq -c '.[] | (if .repo == null then empty else . end ) | .repo' | wc -l)
          # shellcheck disable=SC2003
          # shellcheck disable=SC2086
          total_count=$(expr $tool_count + $repo_count)
```